Automattic/pocket-casts-ios
# Adjust multi-selection header height based on screen size

Fixes #46 

When selecting multiple episodes, the multi-select header overlaps the Search bar on smaller screens (under 380 points). Sample flow for iPhone SE 2nd Gen:

![Test flow - iPhone SE (2nd generation)](https://user-images.githubusercontent.com/81433983/227053920-a2841283-c322-4d60-8aaa-24912db11b75.png)

This PR sets constraint height for multi-select header, based on the size of the display, so that the header does not overlap the search bar. It keeps the constraint height the same as previously set for larger screens.

## To test
Using the workflow above:
1. Open a podcast using a device using a small screen (anything under 380pts width, as [indicated in this chart](https://www.ios-resolution.com/)).
2. Click on Subscribe, if necessary, to view the episode list.
3. Tap the button next to the Search bar to bring up the menu.
4. Tap "Select Episodes" to bring up the multi-select header.
5. Confirm that the header no longer overlaps the Search bar.

Tested on small screens:
- iPod Touch 7th Gen (320pts)
- iPhone SE 2nd Gen (375pts)

![iPod touch - SE 2nd Gen ](https://user-images.githubusercontent.com/81433983/227056632-93cd8dc4-7658-4c88-b1f8-3646662bf9d7.png)

Confirming that the Search bar remains visible on larger screens:
- iPhone 12 Pro (390pts)
- iPhone 14 Pro Max (430pts)

![iPhone 12 Pro - 14 Pro Max](https://user-images.githubusercontent.com/81433983/227058208-d4539a08-6358-454d-b3fc-e2a9474d4434.png)

## Checklist

- [✅] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [✅] I have considered adding unit tests for my changes.
- [✅] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.
